bigquery_etl/events_daily/query_templates/events_daily_v1/init.sql

```diff
@@ -21,8 +21,10 @@ SELECT
   CAST(NULL AS STRING) AS country,
   CAST(NULL AS STRING) AS subdivision1,
   -- normalized fields
+  {% if include_normalized_fields %}
```

Comment:
The normalized fields for `messaging_system.onboarding` don't seem to fill-in correctly. The only one that does is channel. Instead I put this block as optional and include channel explicitly.